pandas/__init__.py

```diff
@@ -46,7 +46,15 @@
 from pandas.tools.merge import (merge, concat, ordered_merge,
                                 merge_ordered, merge_asof)
 from pandas.tools.pivot import pivot_table, crosstab
-from pandas.tools.plotting import scatter_matrix, plot_params
+
+# deprecate tools.plotting, and scatter_matrix on the top namespace
+import pandas.tools.plotting
```

Comment:
you can actually deprecate ``pandas.tools.plotting`` with pandas.util.deprecate_module